src/org/exist/client/InteractiveClient.java

```diff
@@ -1309,18 +1346,15 @@ private synchronized boolean findRecursive(final Collection collection, final Pa
     /**
      * stores given Resource
      *
-     * @param fileName simple file or directory
+     * @param file file or directory
      * @throws XMLDBException
      */
-    protected synchronized boolean parse(String fileName) throws XMLDBException {
+    protected synchronized boolean parse(final Path file) throws XMLDBException {
```

Comment:
`java.nio.file.Paths and java.nio.file.Path are platform aware` - that exactly what I point to you!

for windows result:
```
test/test.file > old > test\test.file
test/test.file > new > test/test.file
file name = test/test.file
test\test.file > old > test\test.file
test\test.file > new > test\test.file
file name = test.file
```

You didn't run test on windows, right?

My point that before scripts did work with "windows paths" on linux & "linux  paths" on windows, but that PR change this behavior.

I don't know why it was coded that way, just point to that fact.